src/components/Proposal/ProposalNavigation.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {useContext, useState, useEffect, useMemo} from 'react';
+import Grid from "semantic-ui-react/dist/commonjs/collections/Grid/Grid"
+import ProposalNavigationItem, {NavigationType} from './ProposalNavigationItem';
+import { UrlParamsContext } from '../Context/UrlParamsContext';
+import useProposals from '../../hooks/useProposals';
+import { useLocation } from '@reach/router'
+import './ProposalNavigation.css';
+
+function ProposalNavigation() {
+
+  const location = useLocation()
+  const p = useMemo(() => new URLSearchParams(location.search), [ location.search ])
```

Comment:
Super minor comment: In general I'd rather have more descriptive names, even if it's an index or something that isn't of our domain

Reply:
fixed!